utils/docker-ce/files/dockerd.init

```diff
@@ -30,17 +93,22 @@ process_config() {
 
 	config_get data_root globals data_root "/opt/docker/"
 	config_get log_level globals log_level "warn"
+	config_get_bool bip_enable globals bip_enable "1"
+	[ "$bip_enable" = "1" ] && config_get bip globals bip "172.17.0.1/24"
```

Comment:
Wouldn't the following do the same?
```
config_get bip globals bip ""
```

Reply:
Yeah, we can do that. But then I would write the default value for the bip option in the uci.